docs/user/exams/students_guide.rst

```diff
@@ -0,0 +1,340 @@
+===================
+Students’ Guide
+===================
+
+General Information
+-------------------
+
+Prerequisites
+^^^^^^^^^^^^^
+- **Stable internet connection**
+
+    **Recommendation:** Use a LAN connection if possible.
+
+- **Browser**
+    
+    **Recommendation:** Chromium (based), e.g. Google Chrome, newest version.
+
+- The following prerequisites are only required if your exam contains programming exercises: 
+
+    - **Java IDE with JDK 15**
+        
+        **Recommendation:** Eclipse IDE 2020‑09.
+    
+    - **Git Client**
+        
+        **Recommendation:** SourceTree
+
+
+Offline Mode
+^^^^^^^^^^^^
+- The exam mode in Artemis tolerates issues with the Internet connection.
+- In case you are not connected, you can continue working on text-, quiz- and modeling exercises, but you might get warnings that your solutions cannot be saved.
+- If your Internet connection recovers, Artemis will save your solution.
+- Artemis tries to save your solution every 30 seconds, when you navigate between exercises and when you click |save| or |save_continue|.
+- Programming exercises have 2 modes.
+
+    1. **Online code editor:** can only be used when you are online. 
+
+        .. note::
+            You have to click on Submit! Otherwise your solution will *not* be pushed to the VC server and no build will be triggered.
+
+    2. **Local IDE:** you only need to be online when you clone the repository and when you push your commits (i.e. submit your solution)
+
+- At the end of the online exercise, you must be online within a given ``grace period`` and submit your exam, otherwise it will not be graded
```

Comment:
```suggestion
- At the end of the online exercise, you must be online within a given ``grace period`` and submit your exam, otherwise it will not be graded.
```